Backpack-SwiftUI/Slider/Classes/BPKRangeSlider.swift

```diff
@@ -100,57 +102,68 @@ public struct BPKRangeSlider: View {
     // swiftlint:disable closure_body_length
     // swiftlint:disable function_body_length
     @ViewBuilder private func sliderView(sliderSize: CGSize) -> some View {
-        ZStack(alignment: .bottom) {
-            Capsule()
-                .fill(Color(.lineColor))
-                .frame(width: sliderSize.width, height: sliderHeight)
-                .padding(.bottom, (thumbSize / 2) - (sliderHeight / 2))
-            Rectangle()
-                .fill(Color(.coreAccentColor))
-                .frame(width: fillLineWidth(sliderSize: sliderSize), height: sliderHeight)
-                .offset(x: fillLineOffset(sliderSize: sliderSize))
-                .padding(.bottom, (thumbSize / 2) - (sliderHeight / 2))
-            SliderThumbView(
-                size: thumbSize,
-                offset: trailingThumbOffset(sliderSize: sliderSize),
-                onDrag: { value in
-                    handleTrailingThumbDrag(value: value, sliderSize: sliderSize)
-                },
-                onDragEnded: { onDragEnded(selectedRange) }
-            )
-            
-            .accessibilityLabel(trailingAccessibilityLabel)
-            .accessibility(value: Text("\(selectedRange.upperBound)"))
-            .accessibilityAdjustableAction { direction in
-                switch direction {
-                case .increment: incrementTrailing()
-                case .decrement: decrementTrailing()
-                @unknown default: break
+        ZStack(alignment: .center) {
```

Comment:
looks massive, but I just wrapped all in this ZStack and moved the thumb labels up one level